sherlock/notify.py

```diff
@@ -5,7 +5,7 @@
 """
 from result import QueryStatus
 from colorama import Fore, Style
-
+globvar = 0 # global variable to count the number of results.
```

Comment:
add a global variable "globvar" to be visible to the whole class and count the number of social medias.